cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/SlowBrokerFinderTest.java

```diff
@@ -44,6 +47,9 @@ public void testDetectingSlowBrokerFromHistory() {
     assertEquals(ANOMALY_DETECTION_TIME_MS, (long) anomaly.entities().get(BROKER_ENTITIES.get(0)));
   }
 
+  /**
+   * Test slow broker finder can detect broker with abnormally high metric in the cluster.
```

Comment:
Is this `consistently abnormally high compared to the other brokers in the cluster`? Does this comparison take excluded brokers for (1) demotion and (2) replica movement?

A broker can be up but does not accept replica moves during its lifetime (this can change dynamically. Do we keep track of its historical changes. If not can we do that to avoid false positives?)

Reply:
A broker which is excluded brokers for demotion and replica movement can still be detected as a  slow broker if it serves large traffic and performs at lower capacity. 
In this situation, self-healing can still move replicas/leadership off this broker.

Reply:
I was thinking about the case, where a `non-excluded` broker is detected as having an abnormally high metric due to other few brokers being excluded; hence, having comparably lower metric.
In this scenario, would excluding such `excluded` brokers from peer comparison be needed?

Reply:
I see, that's a good point!